README.md

```diff
@@ -144,6 +144,13 @@ especially in the case in which the exporter is external to the code it's monito
 
 For more detailed documentation on [Availability exporters](https://gitlab.cee.redhat.com/konflux/docs/documentation/-/blob/main/o11y/monitoring/availability_exporters.md?ref_type=heads)
 
+## Recording Rules
+
+Recording rules allow us to precompute frequently needed or computationally expensive expressions and save their result as a new set of time series. Recording rules are the go-to approach for speeding up the performance of queries that take too long to return. When other teams want to go with their own metrics format for exporters they need to adapt to desired metric form by translating it to the recording rules. 
+These recording rules should be put in the [rhobs/recording folder](https://github.com/redhat-appstudio/o11y/tree/main/rhobs). 
+
+For more detailed documentation on [recording rules](https://docs.google.com/document/d/1Y72T10JGuJaeyeNexmS_qTHfDB8uxxq0zERRRSOZegg/edit?usp=sharing)
```

Comment:
I think we need to have here the requirements for the format that the resulting metric should have. i.e. metric name and the required labels.

It would make most sense to me to have those requirements defined in the same PR as the recording rule example, as then we can compare the requirement and the example and adjust them together.

Reply:
I added the requirement as comment in the recording rule now. Please check if it's sufficient or do I need to specify anywhere else.

Reply:
I think we need the requirements mentioned in some place other than the recording rule. I think this section of the readme file is appropriate.